### PR TITLE
RF | Edit view

### PR DESCRIPTION
**Pull request by olpolezhaeva**
Added Before and AfterMethodes.
Refactor locatores.

### CODE REVIEW

**Comment by olpolezhaeva** [comment]
[RF | Edit view](https://trello.com/c/RlAcrSyy/534-rf-edit-view)

**Review by ksutitova — APPROVED** [review]

File: src/test/java/EditViewChangeNameTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public void createNewView() {
+        getDriver().findElement(By.xpath("//a[@title='New View']")).click();
+        getDriver().findElement(By.id("name")).sendKeys(nameOfView);
+        getDriver().findElement(By.xpath("//label[text()='My View']")).click();
+        getDriver().findElement(By.id("ok")).click();
+    }
+
+    @Test
+    public void testEditView_TC_027_001() {
+        String editNameOfView = String.valueOf((int) (Math.random() * 999));
```
Comment (zenicko) [inline src/test/java/EditViewChangeNameTest.java:21]: Final

File: src/test/java/EditViewChangeNameTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+        WebElement name = getDriver().findElement(By.name("name"));
+        name.clear();
+        name.sendKeys(editNameOfView);
+
+        getDriver().findElement(By.id("yui-gen2-button")).click();
+
+        Assert.assertEquals(editNameOfView,
+                getDriver().findElement(By.xpath("//div[@id='projectstatus-tabBar']//a[contains(@href, '"
+                        + editNameOfView + "')]")).getText());
```
Comment (zenicko) [inline src/test/java/EditViewChangeNameTest.java:35]: low code readability in the lines 34-35

**Review by zenicko — APPROVED** [review]
Errors should never pass silently.

File: src/test/java/EditViewChangeNameTest.java (hunk 2 above)
Comment (olpolezhaeva) [reply src/test/java/EditViewChangeNameTest.java]: Ok, log lines and long Xpath is no good. Moved.

File: src/test/java/EditViewChangeNameTest.java (hunk 1 above)
Comment (olpolezhaeva) [reply src/test/java/EditViewChangeNameTest.java]: Changed both Stringes.

**Pull request merged**